charts/coturn/values.yaml

```diff
@@ -24,6 +24,23 @@ securityContext:
 
 coturnTurnListenPort: 3478
 coturnMetricsListenPort: 9641
+coturnTurnTlsListenPort: 5349
+
+tls:
+  enabled: false
+  certManager:
+    enabled: false
+    dnsNames: []
+    issuer:
+      create: false
+      name:
+      kind:
+      certmasterEmail:
+      solverConfiguration:
+  reloaderImage:
+    repository: quay.io/wire/config-reloader-sidecar
+    pullPolicy: IfNotPresent
+    tag: 1aa6cbbf2ce3a5182ec47e3579bbcb8f47e22fdc
```

Comment:
This could have a comment where it's coming from, and what code it's using.

Reply:
Fixed in [8d5f083](https://github.com/wireapp/wire-server/pull/2620/commits/8d5f0833018c14bf079967b3d18dc0dcac556832)